src/factions/AdeptaSororitas/models.json

```diff
@@ -0,0 +1,465 @@
+                    "SpecialistId": "CO"
+                },
+                {
+                    "SpecialistId": "D"
+                },
+                {
+                    "SpecialistId": "M"
+                },
+                {
+                    "SpecialistId": "SN"
+                },
+                {
+                    "SpecialistId": "V"
+                },
+                {
+                    "SpecialistId": "Z"
+                }
+                ],
+                "ModelProfileWeapons": [
+                    {
+                        "WeaponId": "BO"
+                    },
+                    {
+                        "WeaponId": "PB"
+                    },
+                    {
+                        "WeaponId": "GF"
+                    },
+                    {
+                        "WeaponId": "GK"
+                    }
+                ],
+                "CostOverrides": [],
+                "LevelCosts": []
+            },
+            {
+                "Id": "BAS2",
+                "Movement": 6,
+                "WeaponSkill": 4,
+                "BallisticSkill": 3,
+                "Strength": 3,
+                "Toughness": 3,
+                "Wounds": 1,
+                "Attacks": 1,
+                "Leadership": 7,
+                "Save": 3,
+                "MaximumNumber": 2,
+                "Cost": 10,
+                "NameEn": "Battle Sister Gunner",
+                "NameFr": "",
+                "NameDe": "",
+                "IsCommander": false,
+                "NumberOfKnownPsychics": 0,
+                "NumberOfPsychicsManifestationPerRound": 0,
+                "NumberOfPsychicsDenialPerRound": 0,
+                "WarGearOptions": [
+                    {
+                        "MaximumPerTeam": "0",
+                        "Operation": "BO:(LF|FU|BOST|BOL|HFL)",
+                        "Exclusion": null
+                    }
+                ],
+                "Specialists": [
+                {
+                    "SpecialistId": "CO"
+                },
+                {
+                    "SpecialistId": "D"
+                },
+                {
+                    "SpecialistId": "M"
+                },
+                {
+                    "SpecialistId": "SN"
+                },
+                {
+                    "SpecialistId": "V"
+                },
+                {
+                    "SpecialistId": "Z"
+                },
+                {
+                    "SpecialistId": "AL"
+                }
+                ],
+                "ModelProfileWeapons": [
+                    {
+                        "WeaponId": "BO"
+                    },
+                    {
+                        "WeaponId": "PB"
+                    },
+                    {
+                        "WeaponId": "GF"
+                    },
+                    {
+                        "WeaponId": "GK"
+                    }
+                ],
```

Comment:
Check out utils.Ids I think it's called.

Reply:
👍